node/hyperbahn/hyperbahn.thrift

```diff
@@ -0,0 +1,16 @@
+exception Exception {
+    1: required string message
+}
+
+exception NoHost {
+    1: required string message
+}
+
+service Hyperbahn {
+    list<string> hostportsByService(
```

Comment:
Extensibility in addition to weirdness around how Thrift handles function arguments. Similar in spirit to always returning JSON objects instead of lists.